Author: Dominique Leuenberger <dimstar@opensuse.org>

http://sourceforge.net/tracker/?func=detail&aid=3446968&group_id=239&atid=100239

glib got stricter in checking includes. Only glib.h (and a few exceptions)
are to be included directly.

Index: xchat-2.8.8/src/common/servlist.c
===================================================================
--- xchat-2.8.8.orig/src/common/servlist.c
+++ xchat-2.8.8/src/common/servlist.c
@@ -24,7 +24,7 @@
 #include <unistd.h>
 
 #include "xchat.h"
-#include <glib/ghash.h>
+#include <glib.h>
 
 #include "cfgfiles.h"
 #include "fe.h"
Index: xchat-2.8.8/src/common/text.c
===================================================================
--- xchat-2.8.8.orig/src/common/text.c
+++ xchat-2.8.8/src/common/text.c
@@ -28,7 +28,7 @@
 #include <sys/mman.h>
 
 #include "xchat.h"
-#include <glib/ghash.h>
+#include <glib.h>
 #include "cfgfiles.h"
 #include "chanopt.h"
 #include "plugin.h"
Index: xchat-2.8.8/src/common/util.c
===================================================================
--- xchat-2.8.8.orig/src/common/util.c
+++ xchat-2.8.8/src/common/util.c
@@ -39,7 +39,7 @@
 #include <errno.h>
 #include "xchat.h"
 #include "xchatc.h"
-#include <glib/gmarkup.h>
+#include <glib.h>
 #include <ctype.h>
 #include "util.h"
 #include "../../config.h"
Index: xchat-2.8.8/src/common/xchat.h
===================================================================
--- xchat-2.8.8.orig/src/common/xchat.h
+++ xchat-2.8.8/src/common/xchat.h
@@ -1,10 +1,6 @@
 #include "../../config.h"
 
-#include <glib/gslist.h>
-#include <glib/glist.h>
-#include <glib/gutils.h>
-#include <glib/giochannel.h>
-#include <glib/gstrfuncs.h>
+#include <glib.h>
 #include <time.h>			/* need time_t */
 
 #ifndef XCHAT_H
